Khan/perseus
# Update eslint config to look at package.json at the root for deps

## Summary:
Tamara reported an issue this morning with there being a lot of lint showing up for imports in test files in VSCode.  The reason why this didn't show up in CI is that 'yarn lint' specifies the root .eslintrc.js file.  This prevents cascading of multiple .eslintrc.js files in the file tree.  This PR removes '--config .eslintrc.js' from the 'yarn lint' script and then updates all of the .eslintrc.js files in the packages to include a list of directories to look for package.json files.  We need it to look at both the package.json in the package itself as well as in the root.  The root one contains a bunch of shared dev dependencies such as testing-library and storybook packages.

Issue: None

## Test plan:
- remove a dep from the root package.json
- yarn lint
- see that that extraneous dep is reported
- revert the changes then do the same for a dep in one of the packages' package.json

trivial change to force checks to re-run
